### ISSUE TITLE
Unable to install ViennaRNA on ubuntu

### ISSUE THREAD

**Issue by YonatanGolick** [ViennaRNA/ViennaRNA#177]
Hi, I am unable to install ViennaRNA on a Linux machine.
Getting the following error:
```
Compilation requirements missing!

You deactivated SVM support but this requires the
scripting language interface wrappers (Perl 5, Python)
to be re-generated!

Please either install the 'swig' program to enable
re-generation of the respective wrapper files or:

a) deactivate the scripting language interfaces
   alltogether using the '--without-swig' configure
   option.'
b) leave SVM support enabled by omitting the
   '--without-svm' configure option
```
Tried to run the ./configure with the following options: --without-swig --without-svm but got the same error.
Also I've installed swig on my server but it didn't help as well.
Any ideas ?

**Comment by daz10000** [comment]
I just banged my head against this for several hours trying every combination of flags and environment variables possible.  I have swig, and I didn't try any of those variables.  With / without / swig / svm every combo didn't work.  Finally I just commented out the whole block of the configure script that emits the error message and it ran fine :( - so it's a bug in the configure configuration.

```
#if test "x$with_svm" = "xno"; then
#
#      if test "x$has_swig" != "xyes"; then :
#
#    as_fn_error $? "
#=================================================
#Compilation requirements missing!
#
#You deactivated SVM support but this requires the
#scripting language interface wrappers (Perl 5, Python)
#to be re-generated!
#
#Please either install the 'swig' program to enable
#re-generation of the respective wrapper files or:
#
#a) deactivate the scripting language interfaces
#   alltogether using the '--without-swig' configure
#   option.
#b) leave SVM support enabled by omitting the
#   '--without-svm' configure option
#=================================================
#" "$LINENO" 5

#fi

#fi

```

**Comment by RaumZeit** [comment]
I confirm that there has been a bug in the autoconf instructions when SVM support is deactivated. The fix is already applied to our development branch and will be officially available with the next relase (v2.6.0).
